doc/ref_cert/README.md

```diff
@@ -1,5 +1,41 @@
 [<< Back](../)
 
+# CNTT Reference Conformance (RC) Requirements and Testing Principles
+
+The objective of each RC workstream is to provide an automated mechanism to validate either a network function (NF) or a cloud infrastructure against a standard set of requirements defined by the Reference Model (RM) and associated Reference Architecture (RA).  Through this validation mechanism, a provider of either NFs or cloud infrastructure will be able to test their conformance to the RM and RA.  This will ease the integration of NFs into operator environments that host compatible cloud infrastructures, thereby reducing cost, complexity, and time of integration.
+
+The overall workstream requires the close coordination of the following:
+
+**Requirements** - The agreed upon capabilities and conditions that a compliant NF or cloud infrastructure must provide or satisfy. All requirements will be hosted and maintained in the RA.
+**Tests** - The verification mechanism that determines that a given NF or cloud infrastructure complies with one or more requirements.
+**Conformance Specifications** - The definition of the requirements, tests, and circumstances (test case integration, etc.) that must be met to be deemed conformant.
+
+If there is no clear traceability and strong links between these 3 components, then it becomes difficult to determine if a NF or cloud infrastructure is compliant. With this in mind, below are the set of recommended principles for each of the three components to follow. Adherence to these principles will provide the following:
+
+* Enable clear progress tracking and linkage between independent projects (i.e. know what has and hasn't been covered, and track changes over time)
+* Help users better understand mandatory requirements
+* Provide a stable set of point-in-time requirements and tests to achieve conformance
+* Reduce ambiguity in testing, requirements, and conformance
```

Comment:
I would moderate this part. Even without any traceability, OpenStack testing easily highlights any problem in your infrastructure.The question is more about accepting or not the failure which the traceability may help (e.g. is it about an optional service in CNTT?). If no VM boots in the simplest test cases, we have known we can't onboard any VNF even before CNTT.

Reply:
Do you have a suggestion on what needs to be changed here?

Reply:
It would simply remove "Help users better understand mandatory requirements". I would consider this objective as out of RC (it's more about the referent architecture which defines the requirement as mandatory).

"If there is no clear traceability and strong links between these 3 components, then it becomes difficult to determine if a NF or cloud infrastructure is compliant". Traceability is more about CNTT requirement coverage and somehow project management.

Reply:
edited it slightly to "understand if they meet requirements"